src/pages/Register/Register.scss

```
@@ -0,0 +1,84 @@
+.MuiContainer-root.outer-container {
+  background-color: #112045;
```

```suggestion
  background-color: $some-color
```

Los colores deben estar parametrizados y declarados en el `styles.scss` global. 
No usar colores hardcodeados